recipes/htop/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "htop" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/hishamhm/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 1e93e6743dddad254fac2ad0d83b00a82173c75ecbd6ef6f7d55da690eb64b92
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - automake
+    - libtool
+    - ncurses >=5.9
```

Comment:
This should be `ncurses 5.9*`. Otherwise this will break when `ncurses 6.0` is released here.